amqp/src/test/scala/akka/stream/alpakka/amqp/scaladsl/AmqpConnectorsSpec.scala

```diff
@@ -50,14 +51,16 @@ class AmqpConnectorsSpec extends AmqpSpec {
       //#run-sink
 
       //#run-source
-      val result = amqpSource.map(_.bytes.utf8String).take(input.size).runWith(Sink.seq)
+      val result =
+        amqpSource.map(_.bytes.utf8String).take(input.size).runWith(Sink.seq)
```

Comment:
LOL. There are mobile devices out there, you know? Anyway, any number can be configured.